app/assets/stylesheets/views/_covid.scss

```diff
@@ -51,27 +51,19 @@
   }
 }
 
-.covid__page-header-announcement {
-  margin-bottom: govuk-spacing(7);
-
-  @include govuk-media-query($from: desktop) {
-    margin-bottom: govuk-spacing(3);
-  }
-
-  .govuk-body {
-    margin-bottom: govuk-spacing(2);
-  }
+.covid__announcements {
+  margin-top: govuk-spacing(5);
```

Comment:
You can remove this if you instead modify the heading component at the top of this section to have a margin bottom.